src/ExtensionsSample/Samples/NotificationHubSamples.cs

```diff
@@ -59,8 +80,8 @@ public static class NotificationHubSamples
             await notifications.AddAsync(GetTemplateNotification("World"));
         }
 
-        //   The binding creates a strongly-typed AsyncCollector, which is used to send push notifications. 
-        //   The binding does not do anything with the results when the function exits.  
+        // This binding creates a strongly-typed AsyncCollector, which is used to send push notifications. 
```

Comment:
This method doesn't use AsyncCollector, right? It uses IDictionary. If you're talking about internal implementation details, I don't think that belongs in the sample description.